keel-web/src/main/kotlin/com/netflix/spinnaker/keel/Main.kt

```diff
@@ -47,7 +46,6 @@ private val DEFAULT_PROPS = mapOf(
   PluginsAutoConfiguration::class
 )
 @EnableAsync
-@EnableScheduling
 class KeelApplication
```

Comment:
No longer enable scheduling on the Boot app directly as we want to do it conditionally